src/main/webapp/app/intellij/java-bridge.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { WindowRef } from 'app/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { IntelliJState, JavaDowncallBridge } from 'app/intellij/intellij';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class JavaBridgeService implements JavaDowncallBridge {
```

Comment:
I think this class definitely needs documentation so that other developers are not confused